Kernel/IOWindow.cpp

```diff
@@ -39,19 +39,13 @@ ErrorOr<NonnullOwnPtr<IOWindow>> IOWindow::create_from_io_window_with_offset(u64
     VERIFY(space_type() == SpaceType::Memory);
     VERIFY(m_memory_mapped_range);
 
-// Note: x86-IA32 is the only 32 bit CPU architecture currently being supported and
-// probably will be the only such in the foreseeable future.
-#if ARCH(I386)
-    if (Checked<u32>::addition_would_overflow(m_memory_mapped_range->paddr.get(), offset))
-        return Error::from_errno(EOVERFLOW);
-    if (Checked<u32>::addition_would_overflow(m_memory_mapped_range->paddr.get() + offset, space_length))
-        return Error::from_errno(EOVERFLOW);
-#else
+    // Note: x86-IA64 is the only CPU architecture currently being supported and
```

Comment:
This comment feels less correct when it's a statement about x86_64 than about i386.

Reply:
IA64 also refers to Itanium, not x86-64.

Reply:
> IA64 also refers to Itanium, not x86-64.

This is why I wrote x86-IA64, but I will change it to x86-64.